app/src/main/res/values/environment_config.xml

```diff
@@ -26,7 +26,7 @@
 
     <string name="orcaApiBaseUrl">https://orca.key.app/</string>
 
-    <string name="jupiterQuoteBaseUrl">https://quote-api.jup.ag/</string>
+    <string name="jupiterQuoteBaseUrl">https://swap.keyapp.org/</string>
```

Comment:
what about the prod url? 
When we'll add it?

Reply:
there no prod ulr, we'll discuss it today